test/test_multi_region_poisson_solver.jl

```diff
@@ -63,11 +54,12 @@ function compute_poisson_weights(grid)
     Az = zeros(N...)
     C  = zeros(grid, N...)
     D  = arch_array(grid.architecture, ones(N...))
-    for i = 1:grid.Nx, j = 1:grid.Ny, k = 1:grid.Nz
+    for k = 1:grid.Nz, j = 1:grid.Ny, i = 1:grid.Nx
```

Comment:
This change is very slight.  Can I ask what the motivation is?

Reply:
Thanks!

Reply:
Yep, exactly that. Not that it matters so much here, but I just thought we don't want to "create bad habits" :)

Reply:
And thanks to that I learned something.  Much appreciated.